test/unit/inputType/PanInput.spec.js

```diff
@@ -1,9 +1,8 @@
-import Hammer from "@egjs/hammerjs";
 import {PanInput, getDirectionByAngle, getNextOffset, useDirection} from "../../../src/inputType/PanInput";
```

Comment:
Can't this case make test code?

Reply:
> Well, I actually tried to create a test code for this, but I couldn't reproduce it in our test env where scroll should happen with touch simulations.

I've tested with the existing hammer-simulator, but it didn't work.
Maybe changing Chrome headless to mobile mode with some more fixes can make that work.
But I didn't try that much as I believe the CFC e2e test which I'm working on can do this type of test more easily.